apps/antalmanac/src/lib/enrollmentHistory.ts

```diff
@@ -0,0 +1,158 @@
+import { queryGraphQL } from './helpers';
+import { termData } from './termData';
+
+// This represents the enrollment history of a course section during one quarter
+export interface EnrollmentHistoryGraphQL {
+    year: string;
+    quarter: string;
+    department: string;
+    courseNumber: string;
+    dates: string[];
+    totalEnrolledHistory: string[];
+    maxCapacityHistory: string[];
+    waitlistHistory: string[];
+    instructors: string[];
+}
+
+export interface EnrollmentHistoryGraphQLResponse {
+    data: {
+        enrollmentHistory: EnrollmentHistoryGraphQL[];
+    };
+}
+
+// To organize the data and make it easier to graph the enrollment
+// data, we can merge the dates, totalEnrolledHistory, maxCapacityHistory,
+// and waitlistHistory arrays into one array that contains the enrollment data
+// for each day
```

Comment:
I prefer JSDoc for multiline comments.

```suggestion
/**
  * To organize the data and make it easier to graph the enrollment
  * data, we can merge the dates, totalEnrolledHistory, maxCapacityHistory,
  * and waitlistHistory arrays into one array that contains the enrollment data
  * for each day
 */
```